code/utils/multiple_comparison.py

```diff
@@ -1,3 +1,7 @@
+from scipy import stats
+import numpy as np
+import pdb
+
 """
```

Comment:
the documentation string need to be at the first line of file all the time. Can you change this and push again?
Thanks!